shopgate.php

```diff
@@ -178,6 +178,8 @@ public function install()
             'adminOrder',
             'updateOrderStatus',
             'updateQuantity',
+            'actionValidateOrder',
+            'newOrder'
```

Comment:
Because you register two new hooks in this version, you should add an upgrade file for shops which have already installed your module.